### PR TITLE
remove feature flag for new assignment list, fix bumpy scrolling when paging

### PR DESCRIPTION
**Pull request by bluwave**
refs: MBL-13585
affects: Student
release note: none

use account listed in https://instructure.atlassian.net/browse/MBL-13585 to test, this has a good amount of assignments to test paging.

### CODE REVIEW

**Comment by instructure-cx** [comment]
Scan submitted to Checkmarx

**Comment by instructure-cx** [comment]
### Checkmarx scan completed
[Full Scan Details](https://cxmgr.inssec.info/CxWebClient/ViewerMain.aspx?scanid=1013683&projectid=17)
#### Checkmarx Scan Summary
Severity|Count
---|---
High|0
Medium|0
Low|0
Informational|0

#### Violation Summary
Severity|Count
---|---

**Comment by inst-danger** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Fails</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:no_entry_sign:</td>
      <td>

  Please add a description for Danger.
  </td>
    </tr>
  
<tr>
      <td>:no_entry_sign:</td>
      <td>

  Build failed, skipping coverage check
  </td>
    </tr>
  </tbody>
</table>



<details><summary>:x: Swift lint</summary>


    ⚠️  /Users/vagrant/git/Student/Student/Routes.swift:68:1: Trailing Whitespace Violation: Lines should not have trailing whitespace. (trailing_whitespace)
    Linting 'UIViewControllerExtensionsTests.swift' (90/138)
    Linting 'UIImageStudentExtensionTests.swift' (91/138)
</details>
<details><summary>:x: XCTest failed: StudentUnitTests/AssignmentListViewControllerTests/testPaging</summary>


    Assertion Failure: AssignmentListViewControllerTests.swift:277: XCTAssertEqual failed: ("6") is not equal to ("11")
    Assertion Failure: AssignmentListViewControllerTests.swift:281: XCTAssertEqual failed: ("Optional("Loading...")") is not equal to ("Optional("5")")
</details>
<details><summary>:x: XCTest failed: StudentUnitTests/RoutesTests/testAssignmentList</summary>


    Assertion Failure: RoutesTests.swift:93: XCTAssertEqual failed: ("nil") is not equal to ("Optional("/courses/:courseID/assignments")")
</details>

<p align="right">
  Generated by :no_entry_sign: <a href="http://github.com/danger/danger-js/">dangerJS</a>
</p>

File: Student/Student/Assignments/AssignmentList/AssignmentListViewController.swift [hunk 1]
```diff
@@ -207,18 +216,32 @@ class AssignmentListViewController: UIViewController, ColoredNavViewProtocol, Er
...
-        fetchData()
+        fetchData(showActivityIndicator: true)
     }
 
     func showSpinner(show: Bool = true) {
         if show {
             tableRefresher.beginRefreshing()
-        } else {
+        } else if tableRefresher.isRefreshing {
             tableRefresher.endRefreshing()
```
Comment (bluwave) [inline Student/Student/Assignments/AssignmentList/AssignmentListViewController.swift]: Calling `endRefreshing` was what was causing the scrolling to be choppy while paging.  Don't call this when paging loads are occurring.

**Pull request merged**